src/core/core.tooltip.js

```diff
@@ -505,8 +505,11 @@ module.exports = function(Chart) {
 				tooltipPosition = Chart.Tooltip.positioners[opts.position].call(me, active, me._eventPosition);
 
 				var tooltipItems = [];
-				for (i = 0, len = active.length; i < len; ++i) {
-					tooltipItems.push(createTooltipItem(active[i]));
+				var border = active[0]._xScale.left + 7; // 7 used for graph offset
+				if (existingModel.x > border) {
```

Comment:
I think we should check that `existingModel.x > chartArea.left && existingModel.y < chartArea.right`

Reply:
I think that's a great idea I just keep getting `chartArea` is not defined when I use it. I think chartArea
 only exists in `function determineAlignment(tooltip, size)`. 

![chartarea](https://user-images.githubusercontent.com/27080760/36516577-abd34552-174c-11e8-930a-c5d6bbf8d762.JPG)